src/Microsoft.PowerShell.Commands.Utility/commands/utility/NewGuidCommand.cs

```diff
@@ -0,0 +1,26 @@
+
+/********************************************************************++
+Copyright (c) Microsoft Corporation.  All rights reserved.
+--********************************************************************/
+
+using System;
+using System.Management.Automation;
+
+namespace Microsoft.PowerShell.Commands
+{
+    /// <summary>
+    /// The implementation of the "new-guid" cmdlet
+    /// </summary>    
+    [Cmdlet(VerbsCommon.New, "Guid", HelpUri = "https://go.microsoft.com/fwlink/?LinkId=526920")]
+    [OutputType(typeof(Guid))]
+    public class NewGuidCommand : Cmdlet
+    {      
+        /// <summary>
+        /// returns a guid
+        /// </summary>
+        protected override void EndProcessing()
```

Comment:
>EndProcessing [](start = 32, length = 13)

this implementation subtly changes the current implementation (the function uses Begin rather than End as is here) - I don't believe it to be a big problem, but thought I should add the note